src/components/forms/RangeInput/RangeInput.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { Label } from '../Label/Label'
+
+interface RangeInputProps {
+  id?: string
+  className?: string
+  label?: React.ReactNode
+  hint?: React.ReactNode
+  inputRef?:
+    | string
+    | ((instance: HTMLInputElement | null) => void)
+    | React.RefObject<HTMLInputElement>
+    | null
+    | undefined
+}
+
+export const RangeInput = (
+  props: RangeInputProps & React.InputHTMLAttributes<HTMLInputElement>
+): React.ReactElement => {
+  // Range defaults to min = 0, max = 100, step = 1, and value = (max/2) if not specified.
+  const {
+    id = 'range-slider',
+    name = 'range',
+    className,
+    label,
+    hint,
+    min,
+    max,
+    step,
+    list,
+    inputRef,
+    ...inputProps
+  } = props
+
+  const classes = classnames('usa-range', className)
+
+  return (
+    <>
+      {label && (
```

Comment:
To answer your question, the label should _not_ be included in the component (following the examples of the other input components, which _only_ render an `<input>` element). It should be up to the user to add labels.